### DISCUSSION TITLE
Request to Change Login Email Address

### DISCUSSION THREAD

**Discussion by Cryptonym-R5** [simple-login/app#2194] [Self-hosting question]
I was wondering if it's possible to change the email address I use for logging in. 

Could you let me know how to do that 🤔

**Comment by samueltardieu** [comment]
Isn't the first item in "Account settings" what you're looking for?

![image](https://github.com/user-attachments/assets/47d82a47-4e2a-4b7a-af5a-6ff47aabfa3e)

**Comment by hiccups1980** [comment]
Go here: https://app.simplelogin.io/dashboard/account_setting
And the first thing "account email" is what you want.
